AppCenterAnalytics/AppCenterAnalytics/Internals/MSPropertyConfiguratorPrivate.h

```diff
@@ -39,6 +41,11 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithTransmissionTarget:
     (MSAnalyticsTransmissionTarget *)transmissionTarget;
 
+/**
+ * Indicates if the SDK should collect the "device id" for common schema logs.
+ */
+@property(nonatomic, assign) BOOL shouldCollectDeviceId;
```

Comment:
Move this property definition to the above of initializer.

Reply:
Can we have deviceId string instead of bool flag? Assign a device ID to the variable when `collectDeviceId` is called and use this to be consistent with other property implementations.